lib/components/DateRangeInput.js

```diff
@@ -46,16 +47,17 @@ const DateRangeInput = React.forwardRef(
         )}
         <Input
           ref={ref}
+          allowSingleDayRange={true}
           minDate={minDate}
           maxDate={maxDate}
-          formatDate={(date) => moment(date).format(DATE_FORMAT)}
-          parseDate={(inputStr) =>
-            inputStr
-              ? moment(inputStr, DATE_FORMAT, true).isValid()
-                ? moment(inputStr, DATE_FORMAT, true)
-                : false
-              : null
-          }
+          formatDate={date => moment(date).format(DATE_FORMAT)}
+          parseDate={inputStr => {
+            if (either(isNil, isEmpty)(inputStr)) {
```

Comment:
Can you use R.compose and store in a separate variable `isPresent`. It's done in neetohelp. Please take a look.

Reply:
Sure. Since we only need the negation we can name this `isNotPresent` over here in this component. It makes more sense.